website/pages/docs/plugins/destinations/kafka/overview.md

```diff
@@ -0,0 +1,51 @@
+# Kafka Destination Plugin
+
+import { getLatestVersion } from "../../../../../utils/versions";
+import { Badge } from "../../../../../components/Badge";
+
+<Badge text={"Latest: " + getLatestVersion("destination", "kafka")}/>
+
+This destination plugin lets you sync data from a CloudQuery source to Kafka in various formats such as CSV, JSON. Each table will be pushed to a separate topic.
+
+## Example
+
+This example configures connects to a local Kafka destination with no authentication and pushes messages in JSON format. Note that the plugin only supports `append` write-mode.
+
+The (top level) spec section is described in the [Destination Spec Reference](/docs/reference/destination-spec).
+
+```yaml
+kind: destination
+spec:
+  name: "kafka"
+  path: "cloudquery/kafka"
+  version: "VERSION_DESTINATION_KAFKA"
+  write_mode: "append" # only supports 'append'
+
+  spec:
+    brokers: ["localhost:29092"]
```

Comment:
```suggestion
    brokers: ["<broker-host>:<broker-port>"]
```

Not sure, is the port optional? Can this be a URL, or it has to be <host:port>?

Reply:
`host:port`